content/docs/serverless-workflows/infrastructure/greeting.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Greeting
```

Comment:
nit: Greeting Workflow?

Reply:
> nit: Greeting Workflow?

Other titles in https://github.com/parodos-dev/parodos-dev.github.io/tree/main/content/docs/serverless-workflows do not have "workflow" Let's keep it consistent.